pkg/pillar/Makefile

```diff
@@ -94,7 +94,7 @@ test: build-docker-test
 		--entrypoint /final/opt/gotestsum $(DOCKER_TAG) \
 		--jsonfile /pillar/results.json \
 		--junitfile /pillar/results.xml \
-		--raw-command -- go test -coverprofile=coverage.txt -covermode=atomic -race -json \
+		--raw-command -- go test -tags kubevirt -coverprofile=coverage.txt -covermode=atomic -race -json \
```

Comment:
Is there a unit test that failing to build without the kubevirt tag?
If so, we should put it under `//go:build kubevirt`

Reply:
Not that I know of. This code is given by @christoph-zededa I just used it.

Reply:
There is no unit test that is failing without the kubevirt tag.

There are two reasons I've put it there:
* someone might add test for code that is under the `kubevirt` build flag
* make sure that the current tests are still successful even with the kubevirt build flag set